packages/emotion/src/emotion.ts

```diff
@@ -21,7 +15,7 @@ function createEmotionInstance() {
     key: 'leafygreen-ui',
   };
 
-  if (__TARGET__ === 'web') {
+  if (typeof window !== 'undefined') {
```

Comment:
If we're abandoning the `__TARGET__` build variable, we can update our webpack configuration and package.json files to only output and point to one build file.

Reply:
I think there are still a non-zero number of usages, but they don't impact SSR vs client rendering

Reply:
They don't? The `browser` and `main` fields in the package.json for our components point at different builds that rely on these variables. I'm also not seeing any instances of `__TARGET__` in source code on your branch.

If we simplify to output one build, we could speed up our builds a bit which would be nice.

Reply:
yeah you're right. I was thinking of the storybook fix, which we actually don't need anymore